plugins/GSdx/GSDrawScanlineCodeGenerator.x86.cpp

```diff
@@ -2889,7 +2889,7 @@ void GSDrawScanlineCodeGenerator::ReadTexel_SSE(int pixels, int mip_offset)
 				movdqa(ptr[&m_local.temp.test], xmm7);
 			}
 
-			for(int j = 0; j < 4; j++)
+			for(uint8 j = 0; j < 4; j++)
```

Comment:
Guys, as I said, this loop is never (barely) executed. So let's stop the talk on optimizations.